### DISCUSSION TITLE
Filebeat Module Troubleshooting - Getting Raw Data / Logs

### DISCUSSION THREAD

**Discussion by Git-Me-Some-Hub** [Security-Onion-Solutions/securityonion#6709] [Unsupported Versions]
ISO - 2.90 - Manager - Search Nodes

This may just be a dumb question but I am not sure how I would go about getting raw data that is coming from the Filebeats modules I have been using. I continue to have issue with both my Office 365 and Umbrella Modules built into Filebeats. To troubleshoot I really need to the raw logs that Logstash sees to tell support on either side (Microsoft / Elastic) to see where the problem lives. 

With Syslog I have done TCPdumps, but being this is not really being sent like that I am not sure how I would get the raw data. Any help would be appreciated.

**Comment by dougburks** [comment]
> to tell support on either side (Microsoft / Elastic)

If you're talking to Elastic support, then I would think they would be able to support Logstash and Filebeats.

**Reply by Git-Me-Some-Hub** [reply]
Well, I suppose I should be more specific. I was trying to get some help on their Forums, and found basically nothing. I did have someone reach on Reddit when I posted there, he @ someone on the Elastic team because never did get a response, twice. My post was closed because of inactivity. I was interacting with Microsoft Support, but they would like some type of logs or raw data. I just do not understand why I would get all the Office 365 logs except Exchange. I had followed the documentation both Elastic and Microsoft had.

**Reply by dougburks** [reply]
Perhaps you could configure Filebeat with a `file` output to write to a normal file?
https://www.elastic.co/guide/en/beats/filebeat/current/file-output.html

**Reply by Git-Me-Some-Hub** [reply]
I will check that out, much appreciated.
